src/runtime/components/elements/Feed.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <ol :class="ui.base" v-bind="attrs" :style="ui.style">
+    <template v-for="(item, key) in items" :key="key">
+      <li :class="ui.item">
+        <div>
+          <slot name="icon" v-bind="{ item, key }">
```

Comment:
It could also be nice to handle images here with the `Avatar` component. We could add an `avatar` slot.

Reply:
Seems like a good idea. I think it should just pass the `avatar` props to `UAvatar`.

Reply:
Yeah you can do like everywhere else: https://github.com/nuxt/ui/blob/dev/src/runtime/components/elements/Dropdown.vue#L36

Reply:
Got it.